nixosConfigurations/services/benefice.nix

```diff
@@ -113,8 +113,6 @@ with flake-utils.lib.system; let
 
       profian.environment = "production";
 
-      profian.users.jarkko.enable = true;
```

Comment:
We could have just removed this line if we ever need to give @jarkkojs access to machines